src/components/Admin/index.jsx

```diff
@@ -199,6 +226,42 @@ class Admin extends React.Component {
     return [courseCompletions, enrolledLearners, numberOfUsers].every(item => item === 0);
   }
 
+  renderButtonsRow() {
```

Comment:
nit: A more descriptive name may be `renderSearchBarDownloadBtnRow` or similar. Up to you if you want to change this, but search bar isn't just a button, so having it rendered in a `renderButtonsRow` function seems a bit out of place. Personal preference, though.